### ISSUE TITLE
About the results in tapas paper.

### ISSUE THREAD

**Issue by lairikeqiA** [google-research/tapas#84]
Why the dev accuracy is 29.0%  about the WIKITQ under the full model at table6 in your paper?  Why the dev accuracy is around 50.0%  about the WIKITQ in your GitHub？ What's more, why are the dev accuracy and the test accuracy in your paper so different?

**Comment by eisenjulian** [comment]
Hi @lairikeqiA the numbers of Table 6 of [TaPas: Weakly Supervised Table Parsing via Pre-training](https://www.aclweb.org/anthology/2020.acl-main.398/) are lower than the main results because we in order to run multiple possible pretraining setups in a comparable way, we reduced the number of pretraining steps to 200K (1/5th of the total). This is also explained in the section on **Model ablations**.

As for the best numbers on Github, they are consistent with the paper results when using SQA as an intermediate pre-training task, we also made a further improvement by resetting the position index for every cell (denoted with `_reset` in the model name).

Finally, as for your last question on dev/test discrepancy, can you clarify which numbers you are looking at? I may happen due to the randomness with which the original splits were created.

**Comment by lairikeqiA** [comment]
Thanks!  After you explain the first two questions, the last question does not exist.
